redash/serializers/query_result.py

```diff
@@ -81,13 +82,15 @@ def serialize_query_result(query_result, is_api_user):
 
 
 def serialize_query_result_to_dsv(query_result, delimiter):
-    s = io.StringIO()
+    s = io.BytesIO()
+    s.write(codecs.BOM_UTF8)
```

Comment:
@GunSik2 sorry, I'm still confused. Why do we need to write the BOM manually here if we use UTF-8 later? And why do we need `TextIOWrapper`?